Update threat feed documentation (#52)
phylum-dev/documentation: docs/knowledge_base/threat_feed.md

## Quickstart

> 1. Obtain an API from [the Phylum UI](https://app.phylum.io/tokens/list) and set it as follows:

Reviewer: ```suggestion
1. Obtain an [API key](https://docs.phylum.io/docs/api-keys) and set it as follows:
```

While this may be a little less direct, it might be more robust as CLI methods are added and updated on that page.
